ansible/roles/common/tasks/setup.yml

```diff
@@ -84,4 +84,9 @@
     tags:
       - reboot
   when: not resized.stat.exists
-  
\ No newline at end of file
+  
+# https://github.com/RPi-Distro/firmware-nonfree/blob/bullseye/debian/config/brcm80211/cypress/README.txt#L11-L13
+- name: Setup | Install alternative firmware to maximise the number of clients in AP mode
+  community.general.alternatives:
+    name: cyfmac43455-sdio.bin
```

Comment:
Do we need to download this file first or is it already available on the system?